k-distribution/src/main/scripts/lib/pyk/pyk/kast.py

```diff
@@ -272,32 +272,26 @@ def map_inner(self: 'KApply', f: Callable[[KInner], KInner]) -> 'KApply':
 
 @final
 @dataclass(frozen=True)
-class KAs(KInner, WithKAtt):
+class KAs(KInner):
     pattern: KInner
     alias: KInner
-    att: KAtt
 
-    def __init__(self, pattern: KInner, alias: KInner, att=EMPTY_ATT):
+    def __init__(self, pattern: KInner, alias: KInner):
         object.__setattr__(self, 'pattern', pattern)
         object.__setattr__(self, 'alias', alias)
-        object.__setattr__(self, 'att', att)
 
     @classmethod
     def from_dict(cls: Type['KAs'], d: Dict[str, Any]) -> 'KAs':
         cls._check_node(d)
-        return KAs(pattern=KInner.from_dict(d['pattern']), alias=KInner.from_dict(d['alias']), att=KAtt.from_dict(d['att']) if d.get('att') else EMPTY_ATT)
+        return KAs(pattern=KInner.from_dict(d['pattern']), alias=KInner.from_dict(d['alias']))
 
```

Comment:
This one can probably be tested too. Likely be just trying to read in a frontend definition that uses an `_#as_` pattern somewhere, which can be triggered by using a contextual function.